### PR TITLE
Pm 2020 03 dashboard styling

### PR DESCRIPTION
**Pull request by phillimorland**
@fuzzylogic2000 ok so most of my styling stuff i scrapped anyway so figured would just put it all in same pull request, it's working and not making loads of spans so yey?!

### CODE REVIEW

File: meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html [hunk 1]
```diff
@@ -69,7 +83,7 @@
         {% if project.is_draft and project_progress.valid != project_progress.required or project.published_modules.count == 0 %}
```
Comment (phillimorland) [inline meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html]: I tried to correct this one but my brain is sad, for projects, this doesn't become filled dot when a module is added to project, i'll have another look in morning

File: meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html [hunk 2]
```diff
@@ -13,50 +13,64 @@
...
-        {% elif project.project_type == 'meinberlin_projectcontainers.ProjectContainer' %}
-        <span>{% trans 'Fill in container details' %}</span>
-        {% else %}
-        <span>{% trans 'Fill in project details' %}</span>
+            <i class="fas fa-circle progress-line__icon--active" title="{% trans 'Project not ready for publication' %}" aria-label="{% trans 'Project not ready for publication' %}"></i>
         {% endif %}
+
+
+        <span class="{% for item in dashboard_menu.project %}{{ item.is_active|yesno:"progress-line__entry--active," }}{% endfor %}">
+            {% if project.project_type == 'meinberlin_bplan.Bplan' %}
```
Comment (phillimorland) [inline meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html]: this is how I dealt with it?!

Comment (fuzzylogic2000) [reply meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html]: I think that's a good idea! Will put it into a templatetag with the cleanup, but merge now!

File: meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html (hunk 1 above)
Comment (fuzzylogic2000) [reply meinberlin/templates/a4dashboard/includes/progress_line_in_progress.html]: But this should only become filled when it's possible to publish the project? Oh, yeah, this is not working when a module is created, but not filled in, even if another module was added to the project. It's the same for the publish button, I think. Will ignore for now and have a think later!

**Pull request merged**